### ISSUE TITLE
[a11y] Auth0 "Authorize App" page has unlabelled buttons

### ISSUE THREAD

**Issue by jcsteh** [mozilla-iam/mozilla-iam#81]
When you sign into an app for the first time using Mozilla IAM (e.g. Treeherder, Lando), you are presented with a page asking you to authorise access to your auth tenant. The page has a URL like `https://auth.mozilla.auth0.com/decision?...` and contains something like the following:

```
Authorize App
TREEHERDER.MOZILLA.ORG
Hi James Teh, treeherder.mozilla.org is requesting access to your auth tenant 
Profile: access to your profile and email 
Taskcluster-credentials: taskcluster-credentials 
```

There are then two buttons: one to deny the request and one to allow it. Unfortunately, these buttons have no label for accessibility purposes. Thus, screen reader users (and perhaps users with other accessibility needs) are unable to determine which button is which without DOM inspection or similar. These buttons should be labelled; e.g. using aria-label. The code for the two buttons is as follows:

```
<button id="deny" class="cancel icon icon-budicon-501" value="Deny" name="cancel">
<button id="allow" class="ok icon icon-budicon-500" value="Allow">
```

I am pretty sure this isn't part of the Mozilla "custom lock", which is why I'm filing it here. I'm guessing this is an Auth0 page, in which case I imagine this will have to be filed upstream with Auth0.

**Comment by gdestuynder** [comment]
Hi,

Unfortunately Auth0 currently does not allow customization of the consent dialog (see https://auth0.com/docs/api-auth/user-consent)
> As of today the consent dialog UI cannot be customized or set to a custom domain. We plan to implement this in future releases.

We have requested this feature in the past, and we're currently waiting for it to be implemented (I'm not aware of any ETA other than on the above Auth0 link at this time)

**Comment by jcsteh** [comment]
Understood. Can this accessibility bug be reported to Auth0 so they can fix it on their side? The fix is trivial. I'm happy to answer any questions they might have regarding this.

**Comment by gdestuynder** [comment]
Yes, it has been reported as ticket 47456 (this is not a public ticket) with a reference to this GitHub issue

**Comment by gdestuynder** [comment]
@jcsteh as a quick follow-up: It seems that your suggestion will be implemented relatively soon (I can't guarantee anything of course, and soon could be anywhere from a month to early next year, according to my crystal ball). I'll leave this issue open and will close it when it's actually there.

And thanks for the suggestion!
